### PR TITLE
fix timezone issue

### PR DESCRIPTION
**Pull request by nadarashwin**
## Pull Request Checklist

**Is this in reference to an existing issue?**
https://github.com/sensu-plugins/sensu-plugins-aws/issues/278
#### General

- [ ] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: bin/check-rds-events.rb [hunk 1]
```diff
@@ -117,7 +117,7 @@ def maint_clusters
         # fetch the last 15 minutes of events for each cluster
         # that way, we're only spammed with persistent notifications that we'd care about.
         clusters.each do |cluster_name|
-          events_record = rds.describe_events(start_time: (Time.now - 900).iso8601, source_type: 'db-instance', source_identifier: cluster_name)
+          events_record = rds.describe_events(start_time: (Time.now - 900).strftime('%Y-%m-%dT%H:%M:%SZ'), source_type: 'db-instance', source_identifier: cluster_name)
```
Comment (SegFaultAX) [inline bin/check-rds-events.rb]: I suspect this is not actually what you want. Rather, encode the current UTC time in ISO-8601 format:

```ruby
(Time.now - 900).getgm.iso8601
# => "2018-04-12T16:26:36Z"
```

**Comment by nadarashwin** [comment]
makes sense, thanks @SegFaultAX.
made the required changes.

**Comment by fessyfoo** [comment]
@majormoses thoughts on getting this into a release soon?   to fix #278

**Comment by majormoses** [comment]
Sorry for the delay I have been quite busy with starting the new job hopefully over the next couple weeks things will go back to normal and I will get back to reviewing these regularly.

@nadarashwin thanks for submitting this to sensu plugins project without people like you we could not continue to run the project!

File: bin/check-rds-events.rb [hunk 2]
```diff
@@ -117,7 +117,7 @@ def maint_clusters
         # fetch the last 15 minutes of events for each cluster
         # that way, we're only spammed with persistent notifications that we'd care about.
         clusters.each do |cluster_name|
-          events_record = rds.describe_events(start_time: (Time.now - 900).iso8601, source_type: 'db-instance', source_identifier: cluster_name)
+          events_record = rds.describe_events(start_time: (Time.now - 900).getgm.iso8601, source_type: 'db-instance', source_identifier: cluster_name)
```
Comment (majormoses) [inline bin/check-rds-events.rb]: I was not even aware such a [function](https://ruby-doc.org/core-2.4.0/Time.html#method-i-getgm) existed I generally use [`Time.now.utc`](https://ruby-doc.org/core-2.4.0/Time.html#method-i-utc) which I think is clearer to its intent:
```ruby
irb(main):002:0> Time.now
=> 2018-04-17 18:35:19 -0700
irb(main):003:0> Time.now.utc
=> 2018-04-18 01:35:24 UTC
```

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-aws/versions/11.3.1

**Pull request merged**